.github/workflows/tests.yaml

```diff
@@ -28,8 +30,8 @@ jobs:
     steps:
```

Comment:
I can't comment on the line, but it's above on line 27/28:

Should we bump this to 14.5?